changelog/@unreleased/pr-4555.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: improvement
+improvement:
+  description: AtlasDB now publishes a metric indicating the count of requests that
+    were dropped owing to saturated Cassandra connection pools.
```

Comment:
I don't think connections get dropped, they still retry. Maybe "AtlasDB now publishes a metric that can be used to monitor occurences of Cassandra connection pool exhaustion"

Reply:
Yep, it's a bit confusing - it's the count of requests _internal to the Cassandra client pool_ that are dropped. At a higher level (i.e. `CassandraClientPool` as opposed to the individual pooling container) they will get retried.